client/src/plugins/privacy-preferences/constants.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+export const PRIVACY_TEXT_FIELD = 'To enhance user experience, Camunda Modeler can integrate with 3rd party services, which requires external network requests. Please choose from the settings below.';
+
+export const PRIVACY_POLICY_URL = 'https://camunda.com/legal/privacy/';
```

Comment:
This could actually be read from the env and substituted during the build.

Reply:
I think it's ok to keep it here. Otherwise, we would need to write a new provider to config to provide this env variable to frontend right? Isn't it much easier like this?

Reply:
Let's keep it hard coded for now. We need to make this plugable only once customizing of the update server endpoint becomes a requirement.